app/controllers/items/new.js

```diff
@@ -195,23 +240,25 @@ export default Ember.Controller.extend({
     }
     this.set("locationId", "");
     this.set("inventoryNumber", "");
-    loadingView.destroy();
-    _this.replaceRoute("items.detail", data.item.id);
+    this.hideLoadingSpinner();
+    this.replaceRoute("items.detail", data.item.id);
   },
 
-  getItemConditions(_this) {
+  getItemConditions() {
```

Comment:
Feels like this should be named something like `hasIncompleteConditions` or `isInvalid`
`getXXX` is typically used for getters so a bit confusing : )

Reply:
Would be nice to create a small validation util for models, e.g

```javascript
{
    hasValidConditions: Rule.create([
         { prop: 'description', rules: ['not_empty] },
         { prop: 'code', rules: ['exists'] },
         { prop: 'width', rules: [ ['gt', 0] ] } 
    ]) // => returns an ember computed property
}
```

And use that across the app

Reply:
Yeah.. I have added this name `isInValidConditions` where it has been called. But yes, it would be better to rename it here too.